javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+var $h1 = document.querySelector('h1');
+$h1.textContent = '4';
+function countDown() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+  } else if ($h1.textContent === '1') {
```

Comment:
This is _extremely_ manual. What if you wanted to start the timer at 100? Or 10 thousand? That's a lot of conditional blocks to write. You should make a variable holding a number that would be the text content, then decrement that number every time the function runs. If it reaches zero, you show "earth below us" as the text content